site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 300
```

Comment:
Since the weights are isolated in each drawer would it no be more intuitive to start in each one again with 10 .. n?